.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.20.8
```

Comment:
FWIW if you add `legacy_version_file = yes` to `~/.asdfrc` it will use the current one. Happy to have both 👍🏻

Reply:
🤦 dunno how I missed this, but I already had `legacy_version_file` set and asft seems to be picking it up just fine. I'll remove this file so we're not adding extra junk